vendor/github.com/hinshun/vt10x/expect.go

```diff
@@ -1,8 +1,8 @@
 package vt10x
 
 import (
-	expect "github.com/Netflix/go-expect"
-	"github.com/kr/pty"
+	"github.com/creack/pty"
```

Comment:
wouldn't a glide update or something similar replace and overwrite this?

Reply:
yeah, I remember we decided on forking the `expect` library and changing it there

Reply:
wait..., why would it?

Reply:
oh, interestingly yes. The https://github.com/hinshun/vt10x/blob/master/expect.go which is the upstream package still has `kr/pty` and not `creack/pty`

Reply:
because we have a dependency on the hinshun repo https://github.com/openshift/odo/blob/3ddefa1d0397ba4ed92ecf497a65929d3bf3330b/glide.yaml#L44

Reply:
actually that package is used in `survey` package as well. so I don't prefer to fork 3 packages to solve this problem. That would be too much.

> vt10x package is using the kr/expect package.
> 
> but It seems the github.com/hinshun/vt10x is being used at this place
> 
> https://github.com/openshift/odo/blob/bf899459ce7dfe91964a9f51fcbaa7bf8f64b3c7/pkg/testingutil/survey_ui.go#L10
> 
> 
> and also in this package https://github.com/AlecAivazis/survey.

Reply:
yeah we need to find a better solution :+1:

Reply:
AFAIU ```glide update``` overwrite it. @mik-dass so you want to add the removed repo  ```"github.com/kr/pty"``` back explicitly ?

@girishramnani Question : With the removed repo (Done through glide update) we do not have any build failure then why we need the removed repo back ?

Reply:
@prietyc123 found a solution for this problem https://github.com/openshift/odo/pull/2783/